client/components/recycleAndReuseComponents/VerifyItem.js

```diff
@@ -0,0 +1,169 @@
+                                        <Text width='70%'>{itemToCheckCondition.description}*</Text>
+                                        <Spacer />
+                                        <Select 
+                                            placeholder={selectPlaceholder} 
+                                            onChange={(e) => toggleSelect(e.target.selectedOptions[0].value, index)}
+                                            isRequired={true}
+                                            name={itemToCheckCondition.id}
+                                            value={checkedConditionItems[index]}
+                                        > 
+                                            {itemConditions.map(itemCondition => 
+                                                <option 
+                                                    key={`${itemToCheckCondition.id}-${itemCondition}`}
+                                                    name={itemToCheckCondition.id}
+                                                    value={itemCondition} 
+                                                >
+                                                        {itemCondition}
+                                                </option>
+                                            )}
+                                        </Select>
+                                    </HStack>
+                                )
+                            })}
+                        </VStack>}
+                    </Box>
+                    {invalidItems && showAlert && (
+                        <Alert status='error'>
+                            <VStack width='100%'>
+                                <HStack width='100%'>
+                                    <AlertIcon />
+                                    <Text fontWeight="bold">The following items cannot be recycled.</Text>
+                                    <Spacer />
+                                    <CloseButton onClick={() => toggleShowAlert(false)} size='md' />
+                                </HStack>
+                                <Box width='100%' pl={45}>
+                                    <UnorderedList width='100%'>
+                                        {invalidItems.map((invalidItem) => 
+                                            <ListItem key={invalidItem}>{invalidItem}</ListItem>
+                                        )}
+                                    </UnorderedList>
+                                </Box>
+                            </VStack>
+                        </Alert>
+                    )}
+                    <Button 
+                        size='md' 
+                        colorScheme='teal' 
+                        isDisabled={!enableConfirmButton}
+                        type='submit'
+                    >
+                        Confirm
+                    </Button>
+                </VStack>
+            </form>
+        </Flex>
```

Comment:
It is also possible to refactor this component into multiple child components. Keeping it as one component for now